bitnami/grafana/Chart.yaml

```diff
@@ -24,4 +24,4 @@ name: grafana
 sources:
   - https://github.com/bitnami/bitnami-docker-grafana
   - https://grafana.com/
-version: 4.1.2
+version: 4.1.3
```

Comment:
Hi,
  As you are adding a new parameter to the chart, you should be changing the version to `4.2.0`.
  Do you mind changing it ?